src/components/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
- [ ] [OPTIONAL] Kindly update your MIT license to your project before merging it. Also, do not forget to upgrade your screenshot in your Readme file! Because it seem broken somehow. 

Happy coding!